gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_view.py

```diff
@@ -0,0 +1,1570 @@
+""" pycoingecko_api """
```

Comment:
Apart from TA all of these worked well for me. Amazing work.

The only thing I would have done is add a better description to the commands, so that when the user does "market -h" there is more information on what to expect of type of data, e.g. the rows elements.

Since this is a dataframe, you can use this:
```
", ".join(df.index)
```

on your notebook, and then just copy-paste in there. At least that's how I did it for the Financial Analysis stuff